benchmarks/expected-plans/q11.txt

```diff
@@ -16,7 +16,7 @@
 |               |                 Filter: nation.n_name = Utf8("GERMANY")                                                                                                                         |
 |               |                   TableScan: nation projection=[n_nationkey, n_name]                                                                                                            |
 |               |         SubqueryAlias: __scalar_sq_1                                                                                                                                            |
-|               |           Projection: CAST(SUM(partsupp.ps_supplycost * partsupp.ps_availqty) AS Float64) * Float64(0.0001) AS __value                                                          |
+|               |           Projection: CAST(CAST(SUM(partsupp.ps_supplycost * partsupp.ps_availqty) AS Float64) * Float64(0.0001) AS Decimal128(38, 15)) AS __value                              |
```

Comment:
About merge cast, I prepare to do it in following PR with polish subquery type-coercion